src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball ball1 = new Lottery().getRandomBall(new ColorSupplier().getRandomColor());
+        Ball ball2 = new Lottery().getRandomBall(new ColorSupplier().getRandomColor());
+        Ball ball3 = new Lottery().getRandomBall(new ColorSupplier().getRandomColor());
+        System.out.println(ball1.toString());
+        System.out.println(ball2.toString());
+        System.out.println(ball3.toString());
```

Comment:
1. Lottery should be created once: `Lottery lottery = new Lottery();`
2. find a way to avoid code duplicating - at least we should use loop